### DISCUSSION TITLE
How to reset Admin Password

### DISCUSSION THREAD

**Discussion by switch540** [mediacms-io/mediacms#81] [Q&A] [answered]
I have installed MediaCMS on an Ubuntu 18 x64 standalone server via the script below

mkdir /home/mediacms.io && cd /home/mediacms.io/
git clone https://github.com/mediacms-io/mediacms
cd /home/mediacms.io/mediacms/ && bash ./install.sh

The install appeared to go very smooth, but the admin password that was generated after the installation does not work when trying to login to the web UI.

How do I go about creating a new one or creating a new user account with admin rights?

Thank you

**Comment by mgogoulos** [comment, marked as answer]
Hi,

the installation creates an account with user 'admin' and a random password, not sure why you can't login, are you sure it's not an issue of copy/paste?

Anyway, you can still reset the password through the Django shell - command line

Run as root

```
# source /home/mediacms.io/bin/activate
# cd /home/mediacms.io/mediacms
# python manage.py shell
```

You are in the command line shell. Now run

```
from users.models import User                                                                                                                                                                              
user = User.objects.filter(username='admin')[0]
user.set_password('new_password')
user.save()
```

Then you should be able to login with the combination admin new_password

**Reply by switch540** [reply]
Thank you for the quick reply.  Maybe the install did not go as smooth, as I had thought.  

The following command "python manage.py shell" result is ... "python: can't open file 'manage.py':  [Error 2] No such file or directory.

I am not using copy/paste.  

I have attached a screenshot of the error.  Thank you again for the assistance.

![Python_Error](https://user-images.githubusercontent.com/40492516/110369866-fdc44180-8018-11eb-8d8a-d52829d84530.png)

**Reply by mgogoulos** [reply]
wrong path, you have to be in  `/home/mediacms.io/mediacms` , I've updated the first reply **too**

**Reply by switch540** [reply]
Excellent!  I'm in.  Thank you for the assistance.

**Reply by gam-ragnar** [reply]
@mgogoulos I think the password reset snippet should be added to documentation or solved issues for better visibility.

**Reply by mgogoulos** [reply]
good point

**Reply by Saifhashmi29** [reply]
I have installed mediacms using docker and docker compose in kubuntu 20.04 .. for solving similar problem of django admin login on http://localhost/admin ... i followed above mentioned procedure but faced an error while running the code in python shell ..the error screen shot is attached,,kindly help.
![eror](https://user-images.githubusercontent.com/73126070/122392624-78441c00-cf8d-11eb-984b-d4135b561a4d.png)

**Reply by anasskal** [reply]
Have to get access to the container before taping the recommended commands :

docker ps
docker exec -it <mycontainer> bash
source /home/mediacms.io/bin/activate
cd /home/mediacms.io/mediacms
python manage.py shell

from users.models import User                                                                                                                                                                              
user = User.objects.filter(username='admin')[0]
user.set_password('new_password')
user.save()

Finish by clicking 'Enter' end exit.

**Reply by heerbovec** [reply]
"docker exec -it bash" takes two arguments. One has to provide the name of the container.
